km_api/know_me/factories.py

```diff
@@ -36,6 +36,11 @@ class EmergencyContactFactory(factory.django.DjangoModelFactory):
     Factory for generating ``EmergencyContact`` instances.
     """
     km_user = factory.SubFactory('know_me.factories.KMUserFactory')
+    name = factory.Sequence(lambda n: 'Contact Name {n}'.format(n=n))
+    relation = 'Caregiver'
+    phone_number = '19193334444'
+    alt_phone_number = '19191112222'
+    email = 'knowmetools@knowmetools.com'
```

Comment:
You don't have to specify `alt_phone_number` or `email` since those fields aren't required. It's up to you if you want to include these here, but we don't include optional fields in any of the other factories.